lib/fika/parser/helper.ex

```diff
@@ -147,6 +147,13 @@ defmodule Fika.Parser.Helper do
     {:atom, line, value}
   end
 
+  def join(enumerable, separator \\ "") do
+    Enum.reduce(enumerable, "", fn
+      {:atom, _, value}, acc -> acc <> separator <> ":#{value}"
+      value, acc -> acc <> separator <> value
+    end)
```

Comment:
In my branch, I wrote a parser for atom types that removes the need for this conditional treatment of atoms here. Maybe we can use that?

Reply:
Yeah! I'll try to incorporate that here!

Reply:
I still need a special mapping for atoms here, but only because we need to render the ":" when converting to string :)

Otherwise, atoms are atoms as your PR intended!